docs/sources/storage/_index.md

```diff
@@ -51,10 +64,16 @@ S3 is AWS's hosted object store. It is a good candidate for a managed object sto
 
 ### Notable Mentions
 
-You may use any subsitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
+You may use any substitutable services, such as those that implement the S3 API like [MinIO](https://min.io/).
 
 ## Implementations - Index
 
+### Single-Store
+
+Also known as "boltdb-shipper" during development (and is still the schema `store` name). The single store configurations for Loki utilize the chunk store for both chunks and the index, requiring just one store to run Loki.
+
+As of 2.0, this is the recommended index storage type, performance should be nearly as good as the dedicated index types while providing a much less expensive and less complicated deployment.
```

Comment:
Do we need to say _nearly as good_? It's been better in many scenarios and worse in some, but generally by negligible margins.

Reply:
fixed